data/assets/scene/solarsystem/sun/sun.asset

```diff
@@ -8,7 +8,7 @@ local Sun = {
     Parent = transforms.SunIAU.Identifier,
     Renderable = {
         Type = "RenderableGlobe",
-        Enabled = false,
```

Comment:
this change we want as part of the scene file, not a global change for all scenes.

Reply:
ops